libraries/Microsoft.Bot.Builder.AI.QnA/Models/KnowledgeBaseAnswerSpan.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA.Models
+{
+    /// <summary>
+    /// Gets or sets precise answer of query from Knowledge Base.
+    /// </summary>
+    public class KnowledgeBaseAnswerSpan
```

Comment:
This class seems to be used internally to connect to the service and it is not needed outside, consider making it internal